README.md

```diff
@@ -140,6 +139,38 @@ Options:
 Remove a dependency from a Cargo.toml manifest file.
 ```
 
+### `cargo upgrade`
+
+Upgrade dependencies in your `Cargo.toml` to their latest versions.
+
+#### Examples
+
+```sh
+# Upgrade all dependencies
+$ cargo update
+# Upgrade libc and serde
+$ cargo update -d libc --dependency serde
```

Comment:
I don't think so - the usage syntax is that you need one of `-d`/`--dependency` per dependency to be upgraded. This follows the pattern that `cargo update` uses.